src/model.ts

```diff
@@ -291,7 +291,8 @@ export class Model {
 
 			// Complete pending model initialization steps
 			this._readyProcessing = true;
-			this._readyCallbacks.forEach(init => init());
+			for (const init of this._readyCallbacks)
```

Comment:
This change was unrelated to the fix, but fixes a potential bug with ready callbacks. Array.forEach will not iterate over items added to the array during the iteration, which is a valid scenario when a ready callback queues another callback.